src/pages/index.js

```diff
@@ -111,6 +193,52 @@ profileEditButton.addEventListener("click", () => {
 });
 addNewCardButton.addEventListener("click", () => cardPopupForm.open());
 
+/*=============================================
+=            popup confirmation            =
+=============================================*/
+const deleteConfirmPopup = new PopupWithConfirmation("#confirm-modal");
+deleteConfirmPopup.setEventListeners();
+
+function handleDeleteClick(cardElement) {
+  deleteConfirmPopup.open();
+
+  deleteConfirmPopup.setSubmitAction(() => {
+    deleteConfirmPopup.setLoading(true);
+    api
+      .fetchDeleteCard(cardElement.getId())
+      .then(() => {
+        cardElement.removeCard();
+        deleteConfirmPopup.close();
+      })
+      .catch(console.error)
+      .finally(() => deleteConfirmPopup.setLoading(false));
+  });
+}
+
+/*=============================================
+=            like and dislike            =
+=============================================*/
+
+function handleLikeClick(cardElement) {
+  if (cardElement.getLikes()) {
+    api
+      .fetchDisLikeCard(cardElement.getId())
+      .then((res) => {
+        cardElement.renderLikes(res.isLiked);
+        cardElement._isLiked = res.isLiked;
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.